hms-app/src/pages/Archive.tsx

```diff
@@ -24,12 +89,21 @@ export default function Archive() {
         </>
       )}
 
-      {selectedHackathonId && (
-        <HackathonDetails
-          hackathonId={selectedHackathonId}
-          type={HackathonDetailsType.Archive}
-        />
-      )}
+      {hackathonData.startDate !== NULL_DATE &&
+        hackathonData.startDate?.toString() !== 'Invalid Date' &&
+        !isHackathonLoading &&
+        !isHackathonError && (
```

Comment:
this could probably be extracted to a private function with a name that better explains what it is checking

Reply:
`hackathonData.startDate?.toString() !== 'Invalid Date'` this bit also feels a bit odd to me 🤔

Reply:
about the 'Invalid Date', I'm also not so sure, it was from the HackathonDetails. 
And `hackathonData.startDate?.toString() !== 'Invalid Date'` is also found in a lot of other places (other components).

Do we actually not need it here?